### DISCUSSION TITLE
Update to French locale [fr]

### DISCUSSION THREAD

**Discussion by DenisMo92** [aclap-dev/video-downloadhelper#981] [Language & Translation]
[fr.txt](https://github.com/user-attachments/files/16124750/fr.txt)

**Comment by aaclps** [comment]
[fr.txt](https://github.com/user-attachments/files/16149328/fr.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
